### PR TITLE
Replace `interface{}` with `any`

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
From Spain with Love ™️ 

<sub>https://github.com/golang/go/issues/49884#event-5758961777</sub>

### CODE REVIEW

File: .golangci.yml
Comment (0x2b3bfa0) [inline .golangci.yml:1]: This file only makes sense if https://github.com/martinmunillas/otter/pull/3 gets merged; copied from [open-telemetry/opentelemetry-collector-contrib#29072](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29072/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9)

**Comment by martinmunillas** [comment]
Thank you but this is useless

**Comment by pitasi** [comment]
> Thank you but this is useless

:(

**Comment by 0x2b3bfa0** [comment]
> _Thank you but this is useless_

:(

**Pull request closed without merging**